src/components/curriculum/CurriculumAdvancedReact.js

```diff
@@ -96,22 +96,7 @@ const CurriculumAdvancedReact = ({
       ) : (
         ''
       )}
-      {list ? (
-        <List>
-          {firstHalf}
-          {secondHalf}
-        </List>
-      ) : (
-        <Row>
-          <Col md={6} lg={5} lgOffset={1}>
-            {firstHalf}
-            {secondHalf}
-          </Col>
-          <Col md={6} lg={5} lgOffset={1}>
-            <UpcomingTrainingSection curriculum />
-          </Col>
-        </Row>
-      )}
+      {selectCurriculumLayout(firstHalf, secondHalf, layout)}
```

Comment:
@ric9176 I normally go for an object as argument and deconstruct them so I get named arguments which make function's arguments more flexible. But it's fine with me if you prefer "classic" arguments.

Reply:
:+1: I'll refactor, this is a better approach as in the reactNative there is no secondHalf so I can avoid passing a null value :smile: